.github/workflows/claude.yml

```
@@ -1,67 +1,67 @@
-                skip tests and commit directly.
-              ▸ After pushing, open a pull request that references the original comment.
-
-            After every command, post a short comment summarising what you did
-            (or why you could not comply).
-
-            --- USER COMMENT START ---
-            ${{ github.event.comment.body }}
-            --- USER COMMENT END ---
-
-          allowed_tools: "Comment,View,Bash,RunTests,GlobTool"
+name: Claude PR Assistant
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)